.gitignore

```diff
@@ -3,6 +3,8 @@
 .project
 .sass-cache
 .tags
+.ruby-version
+.ruby-gemset
```

Comment:
remove these lines - both files need to be present, since we require to use same version of ruby